src/components/activity-page/managed-interactive/managed-interactive.test.tsx

```diff
@@ -295,4 +296,68 @@ describe("ManagedInteractive component", () => {
     });
     expect(mockHandleGetAttachmentUrl).toHaveBeenCalled();
   });
+
+  it("renders click to play over the component", async () => {
+    const sampleEmbeddable: EmbeddableType = {
+      name: "mc question",
+      url_fragment: null,
+      authored_state: "{\"version\":1,\"questionType\":\"multiple_choice\",\"multipleAnswers\":false,\"layout\":\"vertical\",\"choices\":[{\"id\":\"1\",\"content\":\"Choice A\",\"correct\":false},{\"id\":\"2\",\"content\":\"Choice B\",\"correct\":false},{\"id\":\"3\",\"content\":\"Choice C\",\"correct\":false}],\"prompt\":\"<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>\",\"hint\":\"<p>this is a hint</p>\"}",
+      is_hidden: false,
+      is_half_width: true,
+      show_in_featured_question_report: true,
+      inherit_aspect_ratio_method: true,
+      custom_aspect_ratio_method: "DEFAULT",
+      inherit_native_width: true,
+      custom_native_width: 576,
+      inherit_native_height: true,
+      custom_native_height: 435,
+      inherit_click_to_play: true,
+      custom_click_to_play: false,
+      inherit_full_window: true,
+      custom_full_window: false,
+      inherit_click_to_play_prompt: true,
+      custom_click_to_play_prompt: null,
+      inherit_image_url: true,
+      custom_image_url: null,
+      library_interactive: {
+        hash: "af5d1860b2d4a037ae01e3d86931a30886fcb42d",
+        data: {
+          aspect_ratio_method: "DEFAULT",
+          authoring_guidance: "",
+          base_url: "https://models-resources.concord.org/question-interactives/branch/master/multiple-choice/",
+          click_to_play: true,
+          click_to_play_prompt: "Click here to play!!!",
```

Comment:
I've looked at the code now to verify that this will work even these properties are up 2 levels on a MwInteractive. But I'd still like a duplicate of this test to make sure that keeps working in the future. Especially since that MwInteractive case is the more common one.

Reply:
I've added a test of the plain MwInteractive in this new commit: https://github.com/concord-consortium/activity-player/pull/355/commits/ac9544eb1a4b60d75df90ec580064ac60d546feb